packages/helloworld/package.json

```diff
@@ -1,40 +1,42 @@
 {
```

Comment:
why this change? is it compatible with the current linter config? btw we should add lint and type checks to the CI pipeline